HTML4FUN

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta content="text/html; charset=utf-8">
+<title> Тестова сторінка  №1.</title>
+</head>
+<body>
+<table width="100%" border="1" cellspacing="1" cellpadding="1" bgcolor="#FFFF00">
+ <tr>
+   <th height="91" colspan=3 align="center"  bgcolor="#66FF00"><strong style="color:red"; font-size:350%; >HTML JUST 4 FUN :)</strong></th>
+ </tr>
+ <tr>
+  <th height="47" colspan=3 align="center" bgcolor="#DC143C">
+      <table width="50%" border="2" cellspacing="1" bgcolor="##FFD700">
+       <tr>
+         <td align="center"><a href="https://github.com/Corvax357" style="text-decoration: none"><font color="#FF6347">Посилання</font></a></td>
+         <td align="center"><a href="http://www.w3schools.com" style="text-decoration: none"><font color="#008080">Друге</font></a></td>
+         <td align="center"><a href="https://www.youtube.com" style="text-decoration: none"><font color="#A0522D">Третє</font></a></td>
+         <td align="center"><a href="http://ololo.fm" style="text-decoration: none"><font color="#FF4500">Четверте</font></a></td>
+       </tr>
+     </table>
+ </th>
+ </tr>
+ <tr>
+ <th width="17%" height="498" align="center" bgcolor="#BFEFFF"> ***DJ.FM***
+  <audio controls> 
+  <source src="relax.mp3" type="audio/mpeg">
+</audio> <br> 
+<h2> Перелік всякої всячини</h2>
```

Comment:
Lets make this page be more topical. Let this page be about auto advertising, for example advertising Subaru WRX STI.